docs/source/top_level_elements/location.rst

```diff
@@ -0,0 +1,208 @@
+
+    <https://example.org/objects/1> relators:rps <http://id.loc.gov/authorities/names/n80003889> .
+
+physicalLocation and shelfLocator
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+https://digital.lib.utk.edu/collections/islandora/object/scopes:1258/datastream/MODS/view
+
+.. code-block:: xml
+
+    <location>
+        <physicalLocation valueURI="http://id.loc.gov/authorities/names/no2014027633">University of Tennessee, Knoxville. Special Collections</physicalLocation>
+        <shelfLocator>Box 5, Folder 8</shelfLocator>
+    </location>
+
+.. code-block:: turtle
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators> .
+    @prefix opaque: <http://opaquenamespace.org/ns/> .
+
+    <https://example.org/objects/1> relators:rps <http://id.loc.gov/authorities/names/no2014027633> ;
+        opaque:locationShelfLocator "Box 5, Folder 8" .
+
+physicalLocation with holdingSimple and holdingExternal
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+https://digital.lib.utk.edu/collections/islandora/object/volvoices:2199/datastream/MODS/view
+
+.. code-block:: xml
+
+    <location>
+        <physicalLocation>University of Memphis. Special Collections</physicalLocation>
+        <holdingSimple>
+            <copyInformation>
+                <shelfLocator>Manuscript Number 5</shelfLocator>
+            </copyInformation>
+        </holdingSimple>
+        <holdingExternal>
+            <holding xsi:schemaLocation="info:ofi/fmt:xml:xsd:iso20775 http://www.loc.gov/standards/iso20775/N130_ISOholdings_v6_1.xsd">
+                <physicalAddress>
+                    <text>City: Memphis</text>
+                    <text>County: Shelby County</text>
+                    <text>State: Tennessee</text>
+                </physicalAddress>
+            </holding>
+        </holdingExternal>
+    </location>
+
+.. code-block:: turtle
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators> .
+    @prefix opaque: <http://opaquenamespace.org/ns/> .
+
+    <https://example.org/objects/1> relators:rps "University of Memphis. Special Collections" ;
+        opaque:locationShelfLocator "Manuscript Number 5" .
+
+physicalLocation with displayLabel="Address"
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+https://digital.lib.utk.edu/collections/islandora/object/arrow:58/datastream/MODS/view
+
+.. code-block:: xml
+
+
+    <location>
+        <physicalLocation>Pi Beta Phi Fraternity</physicalLocation>
+        <physicalLocation displayLabel="Address">1154 Town and Country Commons Drive, Town and Country, Missouri 63017</physicalLocation>
+        <shelfLocator>Box 36, Folder 14</shelfLocator>
+    </location>
+
+*This is one where I'm not sure where to go with.*
+
+.. code-block:: turtle
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators> .
+    @prefix opaque: <http://opaquenamespace.org/ns/> .
+
+    <https://example.org/objects/1> relators:rps "Pi Beta Phi Fraternity" ;
+        opaque:locationShelfLocator "Box 36, Folder 14" .
```

Comment:
URI resolves as:


"Hmmmm...

The vocabulary or term you were looking for doesn't exist."

Reply:
This is a good catch. Samvera's MODS to RDF Mapping Recommendations used this in their recommendations and it caught me. Apparently, I did not read the footnote stating it did not exist yet. 

>  This predicate does not yet exist; see Analysis Process section for more information on OpaqueNamespace.

I will instead suggest that we drop shelfLocator instances in most cases. When absolutely necessary, **boxNumber**, **boxName**, **folderNumber**, and **folderName** are available, see: http://opaquenamespace.org/predicates.